package.json

```diff
@@ -42,13 +42,15 @@
     "opensearch": "node scripts/opensearch",
     "test": "grunt test",
     "test:jest": "node scripts/jest",
+    "test:jest:coverage": "node scripts/jest --coverage",
+    "test:jest:ci": "node scripts/jest --ci --colors --runInBand",
+    "test:jest:ci:coverage": "node scripts/jest --ci --colors --runInBand --coverage",
     "test:jest_integration": "node scripts/jest_integration",
     "test:mocha": "node scripts/mocha",
-    "test:mocha:coverage": "grunt test:mochaCoverage",
+    "test:mocha:coverage": "yarn nyc --reporter=text-summary --reporter=lcov --report-dir=./target/opensearch-dashboards-coverage/mocha node scripts/mocha",
```

Comment:
Do we know that these work in 1.x?

Reply:
When I tried running the above cmd, I got all the tests passing with below summary

```
411 passing (7s)


=============================== Coverage summary ===============================
Statements   : 45.54% ( 1956/4295 )
Branches     : 20.79% ( 431/2073 )
Functions    : 25.82% ( 228/883 )
Lines        : 46.59% ( 1908/4095 )
================================================================================
Done in 16.20s.```